src/react-components/client-info-dialog.js

```diff
@@ -61,26 +65,44 @@ export default class ClientInfoDialog extends Component {
     onClose();
   }
 
-  render() {
+  getPresenceProfile() {
     if (!this.props.presences) return null;
 
     const presence = Object.entries(this.props.presences).find(([k]) => k === this.props.clientId);
     if (!presence) return null;
 
-    const { hubChannel, clientId, onClose } = this.props;
     const metas = presence[1].metas;
-    const meta = metas[metas.length - 1];
-    const displayName = meta.profile.displayName;
-    const title = <div className={styles.title}>{displayName}</div>;
+    return metas[metas.length - 1].profile;
+  }
+
+  componentDidMount() {
+    const { avatarId } = this.getPresenceProfile();
+    if (avatarId) {
+      getAvatarThumbnailUrl(avatarId).then(avatarThumbnailUrl => this.setState({ avatarThumbnailUrl }));
+    }
+  }
+
+  render() {
+    const profile = this.getPresenceProfile();
+    if (!profile) return null;
+
+    const { displayName, communityIdentifier } = profile;
+    const { hubChannel, clientId, onClose } = this.props;
+    const title = (
+      <div className={styles.title}>
+        {displayName}
+        <div className={styles.communityIdentifier}>{communityIdentifier}</div>
```

Comment:
can `communityIdentifier` be null?

Reply:
Yup, it could be null or undefined. But this will just display nothing, as intended.